run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -f ./ircd.yaml ]; then
+  cp oragono.yaml ircd.yaml
+fi
+
+if [ ! -f ircd.db ]; then
+  ./oragono initdb
+fi
+
+if [ ! -f tls.crt ]; then
+  ./oragono mkcerts
+fi
+
+./oragono run
```

Comment:
This should be `exec oragono run`.

This is for a couple reasons. First, and most importantly, it's so signals get propogated to `oragono`.

If `sh` is pid 1 and `oragono` is a child of it, as this does, `sh` will recieve signals (e.g. `SIGTERM` from docker stop, `SIGHUP` for `docker kill -s HUP $container`, etc).... and `sh` isn't a proper init so it won't propogate those forwards.

If `oragono` isn't an appropriate pid1, then a small init program should be used instead.